### PR TITLE
Adding doc on Customizing ROSA Console URL 

### PR DESCRIPTION
**Pull request by FungCharlotte**
Wrote new documentation on how to customize ROSA Consoel URL

### CODE REVIEW

File: content/rosa/customizing-console-route/index.md [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+     rosa list ingress -c <cluster_name>
+     ```
+     > Take note of the default ingress ID.  
+
+1. Enable wildcard policy on the default ingress
+     
+    > Skip this step if you already have wildcard policy enabled on your cluster
+
+     ```bash
+     rosa edit ingress -c <cluster_name> <default-ingress_id> --wildcard-policy WildcardsAllowed
```
Comment (paulczar) [inline content/rosa/customizing-console-route/index.md:80]: why? I don't see anything in this doc that would require a wildcard route ?

File: content/rosa/customizing-console-route/index.md [hunk 2]
```diff
@@ -0,0 +1,123 @@
...
+     ```bash
+     export EMAIL=<YOUR-EMAIL>
+     export DOMAIN=<YOUR-DOMAIN>
+     ```
+
+1. Set remaining environment variables 
+   
+     ```bash
+     export SCRATCH_DIR=/tmp/scratch
+     export ZONE_ID=$(aws route53 list-hosted-zones-by-name --output json \
```
Comment (paulczar) [inline content/rosa/customizing-console-route/index.md:37]: where is this variable being used ?

File: content/rosa/customizing-console-route/index.md [hunk 3]
```diff
@@ -0,0 +1,123 @@
...
+     ```bash
+     rosa edit ingress -c <cluster-id> <default-ingress-id> --cluster-routes-hostname <$DOMAIN> --cluster-routes-tls-secret-ref <tls-secret-created-for-custom-domain>
+     ```
+
+
+   **Important Note**: All routes created on the default ingress will be accessible on .$DOMAIN url
+
+
+### Test
+1. Retrieve ROSA console URL
```
Comment (paulczar) [inline content/rosa/customizing-console-route/index.md:107]: should we have them log out and log back in ?

File: content/rosa/customizing-console-route/index.md (hunk 2 above)
Comment (FungCharlotte) [reply content/rosa/customizing-console-route/index.md:37]: Oops, I will remove the ZONE_ID variable. I don't use that any where in the doc

File: content/rosa/customizing-console-route/index.md (hunk 3 above)
Comment (FungCharlotte) [reply content/rosa/customizing-console-route/index.md:107]: The assumption is that they haven't logged in already, but if they have, then they have to logout and back in. Will add that to the doc. Thank you

**Review by paulczar — APPROVED** [review]

**Pull request merged**